lib/kube/proxy/forwarder.go

```diff
@@ -320,16 +324,32 @@ func NewForwarder(cfg ForwarderConfig) (*Forwarder, error) {
 	router.POST("/api/:ver/namespaces/:podNamespace/pods/:podName/portforward", fwd.withAuth(fwd.portForward))
 	router.GET("/api/:ver/namespaces/:podNamespace/pods/:podName/portforward", fwd.withAuth(fwd.portForward))
 
-	router.GET("/api/:ver/pods", fwd.withAuth(fwd.listPods))
-	router.GET("/api/:ver/namespaces/:podNamespace/pods", fwd.withAuth(fwd.listPods))
 	router.POST("/apis/authorization.k8s.io/:ver/selfsubjectaccessreviews", fwd.withAuth(fwd.selfSubjectAccessReviews))
-	router.DELETE("/api/:ver/namespaces/:podNamespace/pods", fwd.withAuth(fwd.deletePodsCollection))
-	router.POST("/api/:ver/namespaces/:podNamespace/pods", fwd.withAuth(
-		func(ctx *authContext, w http.ResponseWriter, r *http.Request, _ httprouter.Params) (any, error) {
-			// Forward pod creation to default handler.
-			return fwd.catchAll(ctx, w, r)
-		},
-	))
+
+	for k, v := range allowedResources {
+		prefix := "/api"
+		if k.apiGroup != "core" {
+			prefix = filepath.Join("/apis", k.apiGroup)
```

Comment:
Here and below you probably want to use `path.Join`. 

The `filepath` package contains platform-specific behavior, which you don't need here. You always want to use `/` as the separator.

Reply:
in my holy ignorance I thought they were aliases